### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by jma7161**
New last login and previous login feature.

### CODE REVIEW

File: ui/src/App.css [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 #page {
     min-height: calc(100vh - var(--header-height) - var(--footer-height) - var(--page-padding) - var(--page-padding));
-    margin: var(--page-padding) 0 var(--page-padding) 0;
+    margin: var(--page-padding) 0 3em 0;
```
Comment (RogerTheRabbit) [inline ui/src/App.css:9]: Why was this changed?

File: ui/src/components/Tabs/ActionSubmissionsTab/ActionLogs.js [hunk 2]
```diff
@@ -60,10 +60,11 @@ export default function ActionLogs(props) {
                         if (action.mock_id) {
                             submittedBy = `${action.mock_name} (${action.mock_id}) as ${action.name} (${action.system_id})`
                         }
+                        let showNewSubmissionHighlight = new Date(action.submission_datetime) > prevLogin;
                         return (
-                            <TableRow key={idx}>
+                            <TableRow style={{background: showNewSubmissionHighlight? '#7FFFD4' : 'none'}} key={idx}>
                                 {userContext.user?.role !== USERTYPES.STUDENT && <TableCell>{action.display_name || action.title}</TableCell>}
-                                <TableCell>{action.action_title}</TableCell>
+                                <TableCell className="foo">{action.action_title}</TableCell>
```
Comment (RogerTheRabbit) [inline ui/src/components/Tabs/ActionSubmissionsTab/ActionLogs.js:67]: It doesn't look like this CSS does anything. We should remove it

File: ui/src/components/Tabs/StudentsTab/StudentTeamTable.js [hunk 3]
```diff
@@ -70,6 +76,32 @@ export default function StudentTeamTable(props) {
...
             </TableHeaderCell>
         )
+
+        /**
+        * If the user is a student and it is the first table in the students tab, then it will
+        * display last login of teammates. Otherwise it won't, unless you are any other type of user.
+         * This only effects the students tab and the users in the admin tab.
+        **/
+        if(props.isStudent) {
+            console.log(props.firstTable)
```
Comment (RogerTheRabbit) [inline ui/src/components/Tabs/StudentsTab/StudentTeamTable.js]: Should remove this console.log

File: ui/src/components/pages/DashboardPage.js [hunk 4]
```diff
@@ -70,6 +72,12 @@ export default function DashboardPage() {
                         </Tab.Pane>
                     ),
                 },
+            );
+        // Break intentionally left out to take advantage of switch flow
+        // eslint-disable-next-line
+        case "coach":
+            //console.log(user.role);
```
Comment (RogerTheRabbit) [inline ui/src/components/pages/DashboardPage.js:79]: Delete this console.log

File: ui/src/components/shared/allPages/Footer.js [hunk 5]
```diff
@@ -1,82 +1,84 @@
...
+    useEffect(() => {
+        // A user is considered signed in if the user object has a value
+        // This is set when the /whoami endpoint gets hit (currently happening in the Dashboard.js).
+        setSignedIn(Object.keys(user).length !== 0);
+    }, [user]);
 
-    if (history.location.pathname === "/dashboard") {
-        return <></>;
-    }
+    //What is this???
```
Comment (RogerTheRabbit) [inline ui/src/components/shared/allPages/Footer.js:16]: You can just delete this comment

File: ui/src/components/shared/editors/DatabaseTableEditor.js [hunk 6]
```diff
@@ -146,13 +146,19 @@ export default function DatabaseTableEditor(props) {
...
+                                onChange={(value) => {
+                                    handleChange(null, {name : field.name, value : value})
+                                }}
                                 value={formData[field.name]}
-                                onChange={handleChange}
-                                disabled={field.disabled}
+                                labels={us}
+                                // label={field.label}
+                                placeholder={field.placeholder}
+                                //name={field.name}
```
Comment (RogerTheRabbit) [inline ui/src/components/shared/editors/DatabaseTableEditor.js:158]: What are these commented-out fields? Are they not needed? If they're not needed, delete them

File: ui/src/components/shared/editors/DatabaseTableEditor.js [hunk 7]
```diff
@@ -146,13 +146,19 @@ export default function DatabaseTableEditor(props) {
...
+                                //value={formData[field.name]}
+                                //onChange={handleChange}
+                                // disabled={field.disabled}
```
Comment (MaxHoecker) [inline ui/src/components/shared/editors/DatabaseTableEditor.js:161]: ? 👀

**Review by RogerTheRabbit — CHANGES_REQUESTED** [review]

File: ui/src/App.css (hunk 1 above)
Comment (jma7161) [reply ui/src/App.css:9]: fixed the footer problem where there was a white bar at the bottom of the page

File: ui/src/components/Tabs/ActionSubmissionsTab/ActionLogs.js (hunk 2 above)
Comment (jma7161) [reply ui/src/components/Tabs/ActionSubmissionsTab/ActionLogs.js:67]: gone

**Review by RogerTheRabbit — APPROVED** [review]
looks good
